src/containers/Palette/Palette.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 
 const Palette = (props) => {
-  const { palette, setActive } = props
+  const { deletePalette, palette, setActive } = props
   const colors = [palette.color_1, palette.color_2, palette.color_3, palette.color_4, palette.color_5]
+
   return (
     <div className='palette'>
-      <h5>{palette.palette_name}</h5>
+      <h5 onClick={() => setActive(colors)}>{palette.palette_name}</h5>
```

Comment:
I like how you made this easier for users to click on either the palette or the title to set it as the active palette within the generator component.